lib/generators/suspenders/jobs_generator.rb

```diff
@@ -0,0 +1,55 @@
+module Suspenders
+  module Generators
+    class JobsGenerator < Rails::Generators::Base
+      include Suspenders::Generators::APIAppUnsupported
```

Comment:
We can actually configure API only applications to use Sidekiq, since [Active Job](https://edgeguides.rubyonrails.org/active_job_basics.html) does not depend on views. For example, Active Job is responsible for [delivering emails](https://edgeguides.rubyonrails.org/active_job_basics.html#action-mailer), which is something an API only application is capable of doing.

Reply:
Thank you so much for the explanation @stevepolitodesign. 

I meant to write you a message in slack before creating the PR just saying that I was working on this generator by just following the patterns I was seeing in other generators and trying to learn along the way 🙂 but I got carried away.